### DISCUSSION TITLE
How do you change the logo?

### DISCUSSION THREAD

**Discussion by mberni1** [drakkan/sftpgo#909] [Q&A]
I've gone into the "branding" section of the config file and changed the path (and restarted SFTPGo) and I just get empty spaces where the logos should be. This is what I tried:

I set up an FTP instance using this method (I'm running on AWS Lightsail): 
https://aws.amazon.com/premiumsupport/knowledge-center/lightsail-set-up-ftp/

Then I added the logo file I wanted into a new folder. The path I'm using is "/home/ftp_user/ddlogo.png"

Are there specific file dimensions I need to use? What is the path to the logo files so I can just swap it out?

Any other ideas are welcome 😊 

Thanks!

**Comment by drakkan** [comment]
Take a look at the [docs](https://github.com/drakkan/sftpgo/blob/main/docs/full-configuration.md). Expand the configuration file section and search for "branding".

The branding section in this [howto](https://www.howtoforge.com/how-to-install-sftpgo-on-ubuntu-22-04/) might also help

**Reply by mberni1** [reply]
Thanks! The howto helped me find the files. I just had to change permissions on AWS so I could overwrite the existing logo files.

For anyone else wondering, this is the snippet that was most relevant: 

> All the configured paths (favicon_path, logo_path, etc..), must be relative to "static_files_path". 
> On Linux the default "static_files_path" is /usr/share/sftpgo/static, you can create, for example, a "branding" directory and place your custom resources there. 
> For example you can copy your favicon to /usr/share/sftpgo/static/branding/favicon.ico and set the "favicon_path" to /branding/favicon.ico.

Thanks again,
Marc
